tests_e2e/lisa/runbook/azure.yml

```diff
@@ -0,0 +1,82 @@
+name: azure
+extension:
+  - "../testsuites"
+variable:
+  - name: location
+    value: "westus2"
+  - name: subscription_id
+    value: ""
+  - name: resource_group_name
+    value: ""
+    #
+    # Set the vm_name to run on an existing VM
+    #
+  - name: vm_name
+    value: ""
+  - name: marketplace_image
+    value: "Canonical UbuntuServer 18.04-LTS latest"
```

Comment:
looks like this is not in urn format. Is this how we suppose to define images?
How can we pass multiple images? how each scenario override this?

Reply:
We'll define a combinator to pass multiple distros. The LISA documentation has some info on how to do that, but we can always work with them if we need more info or features.